templates/docker-compose.yml.j2

```diff
@@ -25,13 +24,11 @@ services:
 {% endif %}
 {% elif oauth_provider == "google" %}
       --email-domain='{{ oauth_google_domain | mandatory }}'
-{% elif oauth_provider == "keycloak" %}
+{% elif oauth_provider == "keycloak-oidc" %}
```

Comment:
This change will break all the current playbook that use keycloak as provider.
From what i know : Prometheus, Vault, Airbyte test.

Reply:
+ OpenKM

I think it's fine to break the fleets using it - easy to fix.